calico-cloud_versioned_docs/version-3.16/image-assurance/understanding-scan-results.mdx

```diff
@@ -2,111 +2,97 @@
 
-:::note
-
-In the Scan Results and Runtime tabs, the **Registry path** field may be blank if {{prodname}} cannot access this metadata. For example, images from Docker Hub do not specify the registry in the image metadata.
-
-:::
+Scan results for Image Assurance is based on the Common Vulnerability Scoring System v3 (CVSS Scores). The default Pass, Warn, and Fail values will work for the majority of {{prodname}} deployments. However, you may need to change the defaults because of security requirements.
 
-**How Pass, Warn, and Fail scores map to CVSS scores**
+| CVSS v3 scores                              | Image Assurance mapping and defaults | Default settings     |
+|---------------------------------------------|--------------------------------------|----------------------|
+| 0.0 – (None)<br />0.1 – 3.9 (Low)           | Pass = < 3.8                         | Low                  |
+| 4.0 – 6.9 (Medium)                          | Warn = 3.9                           | Medium severity – 7  |
+| 7.0 – 8.9 (High)<br />9.0 – 10.0 (Critical) | Fail = 7.0                           | Critical or high – 8 |
 
-{{prodname}} image scoring is based on the Common Vulnerability Scoring System v3 (CVSS Scores).
-You can change the CVSS score threshold between a Pass and a Warn, and between a Warn and a Fail. Currently, only [CVSS v3 scores](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator) are displayed. CVEs that do not have a CVSS v3 score (too old to have one, or are too new to be assigned one), display a blank score in the UI, and **N/A** in the CLI.
 
-| CVSS v3 scores                              | Image Assurance mapping | Default settings     |
-| ------------------------------------------- | ----------------------- | -------------------- |
-| 0.0 – (None)<br />0.1 – 3.9 (Low)           | Pass = 3.9              | Low                  |
-| 4.0 – 6.9 (Medium)                          | Warn = 7                | Medium severity – 7  |
-| 7.0 – 8.9 (High)<br />9.0 – 10.0 (Critical) | Fail = 8.0              | Critical or high – 8 |
+**Changing the CVSS thresholds**
 
-**Changing the default CVSS threshold values**
+On the right panel, click the **Settings** icon. When changing the CVSS threshold values, note the following:
+- Changes to values take effect immediately, and alter the scan results for images already in the system
+- If you are using admission controller policies, changing a value may allow pods in your Kubernetes cluster that were previously blocked, to now be deployed or vice versa.
 
-The default threshold values (shown below) will work for the majority of {{prodname}} deployments. However, you may need to change the defaults because of security requirements.
+**Why do I have images with `unknown` results**?
 
-![scan-settings](/img/calico-cloud/scan-settings.png)
+If scan results contain `unknowns` it can mean:
+- The Image Assurance scanner is not enabled and running
+- The scanner has not yet reported results yet
```

Comment:
Typo: Remove one of the "yet"'s in "The scanner has not yet reported results yet".

Reply:
Fixed.